modules/reference/pages/feature/feature-overview.adoc

```diff
@@ -48,6 +48,24 @@ The runtime contains default configuration settings so that the configuration yo
 Specify the features that you need, along with any additions or overrides to the default settings, in the `server.xml` file.
 For details about the server configuration, see the xref:config/server-configuration-overview.adoc[server configuration overview].
 
+You can enable a whole set of features all at once using the convenience features. Use these convenience features in the feature manager of your server configuration to add support for your Open Liberty server.
+The following are considered as convenience features for Jakarta 9.1:
+* Jakarta EE Platform 9.1 - To add support for Jakarta EE to your Open Liberty server
+* Jakarta EE Web Profile 9.1 - To add support for Jakarta EE to your Open Liberty server
+* Jakarta EE Application Client 9.1 - To configure an application client component
+* EnterpriseBeans-4.0 - No unique function, only enable other features.
+
+Use the convenience features in the feature manager of your server configuration; for example:
```

Comment:
Enable convenience features by adding them to the `featureManager` element in your  `server.xml` file, as shown in the following example.